app/models/user.rb

```diff
@@ -3,4 +3,6 @@ class User < ActiveRecord::Base
   # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
+  
+  has_many :languages
```

Comment:
You'll need to update these associations now that you've made them more complex.

For some guidance, read through this article from The Odin Project: https://www.theodinproject.com/courses/ruby-on-rails/lessons/active-record-associations